terasoluna-gfw-common-libraries/terasoluna-gfw-codepoints/src/test/java/org/terasoluna/gfw/common/codepoints/validator/ConsistOfValidatorTest.java

```diff
@@ -328,4 +328,94 @@ public void testIsValid_annotation_lastName_is_invalid() throws Exception {
         assertThat(v.getPropertyPath().toString(), is("lastName"));
         assertThat(v.getMessage(), is("not ascii printable!"));
     }
+
+    @Test
+    public void testIsValid_collection_all_valid() throws Exception {
+        Name_Collection name = new Name_Collection(Arrays.asList("ABC",
+                "GHI"), Arrays.asList("ABC", "GHI"));
+        Validator validator = Validation.buildDefaultValidatorFactory()
+                .getValidator();
+        Set<ConstraintViolation<Name_Collection>> violations = validator
+                .validate(name);
+
+        assertThat(violations, is(notNullValue()));
+        assertThat(violations.size(), is(0));
+    }
+
+    @Test
+    public void testIsValid_collection_firstNameList_is_invalid() throws Exception {
+        Name_Collection name = new Name_Collection(Arrays.asList("ＡＢＣ",
+                "GHI"), Arrays.asList("ABC", "GHI"));
+        Validator validator = Validation.buildDefaultValidatorFactory()
+                .getValidator();
+        Set<ConstraintViolation<Name_Collection>> violations = validator
+                .validate(name);
+
+        assertThat(violations, is(notNullValue()));
+        assertThat(violations.size(), is(1));
+
+        ConstraintViolation<Name_Collection> v = violations.iterator().next();
+        assertThat(v.getPropertyPath().toString(), is(
+                "firstNameList[0].<list element>"));
+        assertThat(v.getMessage(), is(
+                "{org.terasoluna.gfw.common.codepoints.ConsistOf.message}"));
+    }
+
+    @Test
+    public void testIsValid_collection_lastNameList_is_invalid() throws Exception {
+        Name_Collection name = new Name_Collection(Arrays.asList("ABC",
+                "GHI"), Arrays.asList("ＡＢＣ", "GHI"));
+        Validator validator = Validation.buildDefaultValidatorFactory()
+                .getValidator();
+        Set<ConstraintViolation<Name_Collection>> violations = validator
+                .validate(name);
+
+        assertThat(violations, is(notNullValue()));
+        assertThat(violations.size(), is(1));
+
+        ConstraintViolation<Name_Collection> v = violations.iterator().next();
+        assertThat(v.getPropertyPath().toString(), is(
+                "lastNameList[0].<list element>"));
+        assertThat(v.getMessage(), is(
+                "{org.terasoluna.gfw.common.codepoints.ConsistOf.message}"));
+    }
+
+    @Test
+    public void testIsValid_collection_all_is_invalid() throws Exception {
+        Name_Collection name = new Name_Collection(Arrays.asList("ＡＢＣ",
```

Comment:
- [ ] Full widths `ＡＢＣ`??? Should set alphabet not `AtoF` `GtoL`.

Reply:
5a1018380f486608e69af557adc64758ecc75130
I fixed at above commit .